tasks/configuration.yml

```diff
@@ -104,7 +94,7 @@
 
 - name: Copy script
```

Comment:
A more appropriate task name would be "Copy additional SQL queries" don't you think?